src/Rules/Namespaces/ExistingNamesInGroupUseRule.php

```diff
@@ -73,7 +73,7 @@ private function checkConstant(Node\Name $name): ?IdentifierRuleError
 	{
 		if (!$this->reflectionProvider->hasConstant($name, null)) {
 			return RuleErrorBuilder::message(sprintf('Used constant %s not found.', (string) $name))
-				->discoveringSymbolsTip()
+				->discoveringSymbolsTip((string) $name)
```

Comment:
Please remove the passed argument in cases where it's not a class name.

Reply:
So don't do what you asked me to do in the issue?

> Yes, all 22 calls should be updated.

Reply:
(I did ask about non-classes in a comment on the issue.)

> One more clarification. I assume that although we're calling the new argument $className, what we want to test is the name of the thing which was not found, even if that "thing" isn't a class (for example, an interface, contstant, trait, etc.). Right?